tools/execsnoop.py

```diff
@@ -207,6 +208,7 @@ def parse_uid(user):
     // as the real_parent->tgid.
     // We use the get_ppid function as a fallback in those cases. (#1883)
     data.ppid = task->real_parent->tgid;
+    data.cpu = task->cpu;
```

Comment:
Which kernel version you are testing? I checked latest upstream and 5.19, the field name for 'task' is 'on_cpu' and not 'cpu'. Please ensure your change works for all supported kernels (from e.g. 4.11 to recent kernels).

Reply:
Yes, 'cpu' filed was changed on recent kernels. It was move back to 'thread_info' since linux 5.16
Link: https://github.com/torvalds/linux/commit/bcf9033e5449
Now, BTF filed check or buildtime filed check is added to handle this change.